.github/workflows/dot-org-filter.yml

```diff
@@ -0,0 +1,41 @@
+name: Anaconda.org Isuse Notification to Slack
+
+on:
+  issues:
+    types: [opened, edited, labeled]
+
+jobs:
+  check-conditions:
+    if: contains(github.event.issue.labels.*.name, 'type-bug')
+    runs-on:
+      labels: self-hosted-main
+    outputs:
+      should-notify: ${{ steps.filter.outputs.should-notify }}
+    steps:
+    - name: Check if "Anaconda.org" checkbox is checked
+      id: filter
+      run: |
+        body="${{ github.event.issue.body }}"
+        search_term="- [x] Anaconda.org"
+
+        if echo "$body" | grep -Fxq "$search_term"; then
+          echo "::set-output name=should-notify::true"
+        else
+          echo "::set-output name=should-notify::false"
+        fi
+
+  notify-slack:
+    if: needs.check-conditions.outputs.should-notify == 'true'
+    runs-on:
+      labels: self-hosted-main
+    needs: [check-conditions]
+    steps:
+    - name: Send message to Slack
+      uses: rtCamp/action-slack-notify@v2
```

Comment:
this is an external extension, so I'm looking for what we typically use. I noticed some more complex flows with bots.